### PR TITLE
Update preferences alert content

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes update the preferences alert content (https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15526). 

## Testing done
-verified locally

## Screenshots
Save succeeded
screen shot 2018-12-11 at 10 25 56 am

Save failed
screen shot 2018-12-11 at 10 24 45 am

Retrieve failed
screen shot 2018-12-11 at 11 26 09 am

Save failed
screen shot 2018-12-11 at 11 31 50 am

## Acceptance criteria
- [x] Update the preferences alert content.

## Definition of done
~- [ ] Events are logged appropriately~
~- [ ] Documentation has been updated, if applicable~
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by Samara-Strauss** [comment]
Screenshots look good to me!

File: src/applications/personalization/preferences/helpers.jsx [hunk 1]
```diff
@@ -245,3 +246,40 @@ export const benefitChoices = [
...
+      </span>
+      <br />
+      <span>Please try again or check back later.</span>
+    </div>
+  </AlertBox>
+);
+
+export const SaveSucceededMessageComponent = (
+  <AlertBox status="success" headline="We’ve saved your preferences." />
+);
```
Comment (ncksllvn) [inline src/applications/personalization/preferences/helpers.jsx]: Should the "If you're at risk of becoming homeless" warning be here as well?

File: src/applications/personalization/preferences/helpers.jsx [hunk 2]
```diff
@@ -245,3 +246,40 @@ export const benefitChoices = [
...
+        We’re sorry. Something went wrong on our end, and we couldn’t save your
+        update.
+      </span>
+      <br />
+      <span>Please try again or check back later.</span>
+    </div>
+  </AlertBox>
+);
+
+export const SaveSucceededMessageComponent = (
```
Comment (ncksllvn) [inline src/applications/personalization/preferences/helpers.jsx]: Does this one need a `onCloseAlert` prop?

**Review by ncksllvn — APPROVED** [review]
LGTM, just a couple questions. I don't know what the consuming code is like though so I'm not totally sure.

File: src/applications/personalization/preferences/helpers.jsx (hunk 2 above)
Comment (cehsu) [reply src/applications/personalization/preferences/helpers.jsx]: Good catch, added.

File: src/applications/personalization/preferences/helpers.jsx (hunk 1 above)
Comment (cehsu) [reply src/applications/personalization/preferences/helpers.jsx]: Good idea, it happens to be on line 118.

**Pull request merged**